1.1/spec/21-Annex-F.adoc

```diff
@@ -0,0 +1,225 @@
+geo:sfIntersects())`` | ``FILTER(geo:sfIntersects(?geom, "POLYGON( (160.6 -55.95,160.6 -25.89, -170 -25.89, -170 -55.95, 160.6 -55.95))"^^geo:wktLiteral)``
+) | WKT does not define a type boundingbox, therefore a bbox is converted to a Polygon
+| datetime | ``datetime= 2018-02-12 T23%3A20%3A52Z`` | - | - | GeoSPARQL doesn't detail temporal aspects of data. Filtering data using RDF temporal properties may be achieved using basic SPARQL queries and also OWL TIME <<TIME>>
+|===
+
+
+=== F.2.2 Literal Values
+
+CQL2 defines literal values for a variety of datatypes. The following table shows the equivalences of these values in RDF which may be used in any GeoSPARQL query.
+
+|===
+| CQL2 literal | Examples  | (Geo)SPARQL literal | Examples
+
+| String  | "This is a string" | ``xsd:string`` | "This is a string"^^xsd:string 
+| Number  | -100  3.14159 | ``xsd:int``, ``xsd:integer``, ``xsd:double`` | "-100"^^xsd:integer  "3.14159"^^xsd:double 
+| Boolean  | true false | ``xsd:boolean`` | "true"^^xsd:boolean  "false"^^xsd:boolean 
+| Spatial Geometry (WKT)  | POINT(1 1) | <<RDFS Datatype: geo:wktLiteral, WKT Literal>> | "POINT(1 1)"^^geo:wktLiteral
+| Spatial Geometry (JSON)  | {"type": "Point", "coordinates":[1,1]} | <<RDFS Datatype: geo:geoJSONLiteral, GeoJSON Literal>> | "{"type": "Point", "coordinates":[1,1]}"^^geo:geoJSONLiteral
+| Temporal Literal  | 1969-07-20  1969-07-20T20:17:40Z | ``xsd:date``, ``xsd:dateTime``, ``xsd:dateTimeStamp`` | "1969-07-20"^^xsd:date "1969-07-20T20:17:40Z"^^xsd:dateTime
+|===
+
+=== F.2.3 Property references
+
+CQL2 allows the referencing of properties in a Feature Collection it is targeting for filtering. A property reference is converted to a triple pattern as shown in the following example. A SPARQL variable ``?item`` is assumed to represent the Feature Collection.
+
+|===
+| Property Reference  | Triple pattern
+
+| name="OGC" | ``?item my:name "OGC"^^xsd:string``
+| number=5 | ``?item my:number "5"^^xsd:integer``
+| number>5 | ``?item my:number ?number . FILTER(?number>5)``
+|===
+
+=== F.2.4 Comparison Predicates
+
+CQL2 defines comparison predicates to compare two scalar expressions. A comparison predicate is converted to a triple pattern as shown in the following example. A SPARQL variable `?item` is assumed to represent the Feature Collection.
+
+|===
+| Comparison predicate  | Triple pattern | Comment
+
+| name="OGC" | `?item my:name "OGC"^^xsd:string` | Equality statements can be converted to a triple pattern
+| number=5 | ``?item my:number "5"^^xsd:integer`` | 
+| number>5 | ?item my:number ?number . FILTER(?number>5) | Arithmetic comparisons (<,>,>=,<=) are converted to filter expressions
+| number BETWEEN 5 AND 10 | `?item my:number ?number . FILTER(?number>=5 && ?number<=10)` | BETWEEN statements are converted to arithmetic expressions
+| name IN ("OGC","W3C") | `VALUES ?namevalues {"OGC" "W3C"} ?item my:name ?namevalues` | IN statements are expressed by VALUES statements
```

Comment:
Is there any reason not to convert CQL IN to SPARQL IN? VALUES also works. Just curious.

Reply:
I've flipped it to use `IN` and indicate `VALUES` can be used as I agree this is a more natural/direct mapping!